AdaptiveCardsDemo/AdaptiveCardsDemo/ViewController.swift

```diff
@@ -8,10 +8,13 @@
 import UIKit
 import SafariServices
 
-class ViewController: UIViewController {
 
+class ViewController: UIViewController {
+    var resolvers = ACOResourceResolvers.init();
     //MARK: - Lifecycle methods
     override func viewDidLoad() {
+        let resolver = ADCResolver.init();
+        resolvers?.setResourceResolver(resolver, scheme: "https")
```

Comment:
you should also add all schemes you intend to support such as "http" and "data".

Reply:
@nesalang, Thanks for swift response.
Also, it would be great if you can provide link for MSFT's Sample code for AdaptiveCards.

Reply:
@hemantgore 
https://github.com/microsoft/AdaptiveCards/blob/master/source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m
it's LN 121-123